lightning/src/ln/msgs.rs

```diff
@@ -1172,6 +1173,7 @@ impl Readable for Init {
 		let features: InitFeatures = Readable::read(r)?;
 		Ok(Init {
 			features: features.or(global_features),
+			net_address: None,
```

Comment:
You'll want to implement reading and writing `net_address` by using `encode_tlv_stream` and `decode_tlv_stream` at the end of the writeable/readable implementations here.